tests/test_pii_protection.py

```
@@ -0,0 +1,179 @@
+                'financial_general_ledger': {'path': ''},  # Empty path
+                'financial_cogs': {'path': ''}  # Empty path
+            }
+            
+            extractor = FinancialExtractor(config)
+            
+            # Mock FileUtils.find_files to track if it's called
+            with patch('etl_pipeline.extract.financial_extractor.FileUtils.find_files') as mock_find_files:
+                result = extractor.extract()
+                
+                # Should not call find_files with empty paths
+                assert mock_find_files.call_count == 0
```

Ruff (0.12.2)

Use of `assert` detected

(S101)